<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HS device calibration tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="slitAngle_tab">
       <attribute name="title">
        <string>Slit angle</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QGraphicsView" name="slitAngle_graphicsView">
          <property name="backgroundBrush">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <widget class="QSlider" name="slitAngle_horizontalSlider">
            <property name="minimum">
             <number>-89900000</number>
            </property>
            <property name="maximum">
             <number>89900000</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="slitAngle_label">
            <property name="text">
             <string>Slit angle</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QDoubleSpinBox" name="slitAngle_doubleSpinBox">
            <property name="decimals">
             <number>6</number>
            </property>
            <property name="minimum">
             <double>-89.900000000000006</double>
            </property>
            <property name="maximum">
             <double>89.900000000000006</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="slitIntercept_label">
            <property name="text">
             <string>Slit intercept</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSlider" name="slitIntercept_horizontalSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="slitIntercept_doubleSpinBox">
            <property name="decimals">
             <number>6</number>
            </property>
            <property name="minimum">
             <double>-89.900000000000006</double>
            </property>
            <property name="maximum">
             <double>89.900000000000006</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <widget class="QLineEdit" name="slitImagePath_lineEdit"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="slitImagePath_label">
            <property name="text">
             <string>Slit image</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="slitImagePathOpen_pushButton">
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="id" stdset="0">
             <string>open</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="calcAngle_pushButton">
            <property name="text">
             <string>Calc angle</string>
            </property>
            <property name="id" stdset="0">
             <string>big</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="barrelDistortion_tab">
       <attribute name="title">
        <string>Barrel distortion</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="wavelengths_tab">
       <attribute name="title">
        <string>Wavelengths</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="lightMask_tab">
       <attribute name="title">
        <string>Light mask</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="fileOpen_action"/>
    <addaction name="fileExit_action"/>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="helpAbout_action"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="help_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="fileOpen_action">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="fileExit_action">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="helpAbout_action">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
